src/components/navigation/navigation.tsx

```diff
@@ -15,15 +16,12 @@ export interface NavigationProps {
 }
 
 const Navigation = ({ children }: NavigationProps) => {
-    return (
-        <div className="flex w-[180px] grow flex-col overflow-y-auto bg-primary-900 pb-5 pt-3">
-            {children}
-        </div>
-    );
+    return <div className="flex w-[180px] grow flex-col bg-primary-900 pb-5 pt-3">{children}</div>;
```

Comment:
Not sure if this was intended to remove the `overflow-y-auto` but do you test it in smaller screens? If the items in the navigation grows or screen size height is too small, we need to ensure that the scroll is applied.

Reply:
It was necessary to allow the popover to be seen. I've some tests, and it doesn't seem to have any negative effect.